src/commands/index.ts

```diff
@@ -270,6 +271,7 @@ const commands: CommandDefinition[] = [
     dlss,
     temporarycommandedit,
     temporarycommand,
+    yourcontrols,
```

Comment:
```suggestion
    yourControls,
```

We will be refactoring to use camelCase at some point, so starting now eases our pain later haha!